torch/backends/_nnapi/serializer.py

```diff
@@ -1230,9 +1236,14 @@ def _do_add_binary(self, node, opcode, fuse_code, *, qparams=None):
         assert node.inputsAt(0).type().kind() == "TensorType"
         assert node.inputsAt(1).type().kind() == "TensorType"
 
-        # TODO: Should support constant as either operand.
-        in0_id, in0_oper = self.get_tensor_operand_by_jitval(node.inputsAt(0))
-        in1_id, in1_oper = self.get_tensor_operand_by_jitval(node.inputsAt(1))
+        if self.has_operand_for_jitval(node.inputsAt(0)):
+            in0_id, in0_oper = self.get_tensor_operand_by_jitval_fixed_size(node.inputsAt(0))
```

Comment:
Does this need to be fixed size?  I think I had this in my version because I'm based on a revision where we're forcing fixed size here.